test/scripts/check_max_storage_in_orthanc_raw.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+This script will upload additional instances to orthanc-raw and then confirm
+that the original test images have been removed.
+
+Polling to allow for orthanc processing time.
+"""
```

Comment:
I think it'd be useful to document more fully what approach the test is taking here. e.g. include that the additional instances are intended to fill up the storage, as well as cause the original test images to be overwritten.

Reply:
Thanks, I've expanded that docstring a bit.